src/app/tl-header/tl-header.component.scss

```diff
@@ -27,7 +33,13 @@
       --wvr-nav-link-background-hover: var(--tl-deep-grey);
     }
 
+    wvr-nav-list-element[bottom-navigation] wvr-nav-li {
```

Comment:
wvr-nav-list-element[bottom-navigation] wvr-nav-li  - this block is appearing twice.

Reply:
Fixed it.